mesheryctl/internal/cli/root/system/model.go

```diff
@@ -356,6 +356,7 @@ mesheryctl exp model view [model-name]
 	RunE: func(cmd *cobra.Command, args []string) error {
 		if len(args) == 0 {
 			_ = cmd.Help()
```

Comment:
return err?

Reply:
showing help message when no arguments are passed, so `return nil` will be viable.